invenio_drafts_resources/resources/records/config.py

```diff
@@ -21,3 +24,19 @@ class RecordResourceConfig(_RecordResourceConfig):
     draft_links_config = {
         "record": DraftLinksSchema
     }
+
+class RecordVersionsResourceConfig(_RecordResourceConfig):
+    """Record resource version config."""
+
+    list_route = "/records/<pid_value>/versions"
+
+    item_route = "/records/<pid_value>/versions/latest"
```

Comment:
Because we inherit from `CollectionResource` we need to define `item_route` or override the `create_url_rules` function and return only the list_route. But, I thought that the item_route can be used for displaying the latest version, so we don't have to create yet another resource...